src/components/Post/Post.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Post.scss';
+import { CommentList } from '../CommentList';
+import { User } from '../User';
+
+export const Post = ({ title, body, comments, user }) => (
+  <>
+    <div className="post">
+      <div className="postBlock">
+        <div>
+          <h2 className="postName">{title}</h2>
+          <p className="postBody">{body}</p>
+        </div>
+        <User {...user} />
+      </div>
+    </div>
+
+    <CommentList commentss={comments} />
```

Comment:
```suggestion
    <CommentList comments={comments} />
```
or those double 's' is your style?)